client/blocks/signup-form/index.jsx

```diff
@@ -704,7 +708,9 @@ class SignupForm extends Component {
 		return (
 			<LoggedOutFormLinks>
 				<LoggedOutFormLinkItem href={ logInUrl }>
-					{ this.props.translate( 'Already have a WordPress.com account?' ) }
+					{ [ 'onboarding', 'onboarding-dev' ].includes( flowName )
```

Comment:
Curious why we would only show this for the onboarding flows and not the others as well?

Reply:
> Curious why we would only show this for the onboarding flows and not the others as well?

If someone is seeing this block e.g. via the Gravatar flow where they're not creating a site, the string `Already have a WordPress.com account?` makes more sense to me.